bash_scripts/run_sim.sh

```diff
@@ -1,40 +1,68 @@
 #!/bin/bash
 
-#SBATCH -p cpu-long
-#SBATCH -c 1
+#SBATCH -p arm-preempt
+#SBATCH -c 20
 #SBATCH -G 0
-#SBATCH --mem=16000
-#SBATCH -t 1-00:00:00
-#SBATCH -o slurm-%j.out
+#SBATCH --mem=40000
+#SBATCH -t 14-00:00:00
+#SBATCH -o slurm-%A_%a.out  # Output file for each task
+#SBATCH --array=0-3  # Define array size based on total combinations
 
-# This script is designed to run a non-artificial intelligence simulation on the Unity cluster at UMass Amherst.
-# Users can provide custom parameters via the command line using -- before the parameter list.
+set -e  # Stop the script if any command fails
 
-# Ensure the script stops if any command fails
-set -e
+echo "Starting job with SLURM_ARRAY_TASK_ID: $SLURM_ARRAY_TASK_ID"
 
-# Change to the home directory and then to the SDN simulator directory
```

Comment:
We should take this as command line input but have a default to this directory. It's not clear where the simulation would run for the user.